var/spack/repos/builtin/packages/py-ruff/package.py

```diff
@@ -30,6 +31,7 @@ class PyRuff(PythonPackage):
         depends_on("py-maturin@1")
 
         # Found in Cargo.toml
+        depends_on("rust@1.83", when="@0.9.8:")
```

Comment:
```suggestion
        depends_on("rust@1.83:", when="@0.9.8:")
```

@cmelone is the dependency on Rust v1.83 **only**, a requirement or does v0.9.8 bump the minimum?

Reply:
good catch, thanks